app/jobs/sms_sender_confirmation_job.rb

```diff
@@ -4,7 +4,7 @@ class SmsSenderConfirmationJob < ActiveJob::Base
   def perform(code, mobile)
     TwilioService.new.send_sms(
       to: mobile,
-      body: "Your #{APP_NAME} phone confirmation code is: #{code}."
+      body: "#{code} is your #{APP_NAME} one-time passcode."
```

Comment:
Seems like either way would be fine as long as the actual code doesn't get cut off from the initial popup notification you get on your phone.  Otherwise you are forced to unlock your phone and go pull up the message in order to view the entirety of it which can be frustrating.  I would vote for the compromise of `PayPay: 123445 is your confirmation code`.